External/FEXCore/Source/Interface/Core/JIT/Arm64/JIT.cpp

```diff
@@ -422,8 +420,10 @@ void *JITCore::CompileCode([[maybe_unused]] FEXCore::IR::IRListView<true> const
         uint64_t SPOffset = AlignUp((RA64.size() + 7 + 1) * 8, 16);
 
         sub(sp, sp, SPOffset);
-        for (uint32_t i = 0; i < 7; ++i)
+        for (uint32_t i = 0; i < 7; ++i) {
```

Comment:
Why not `FEXCore::HLE::SyscallArguments::MAX_ARGS`?

Reply:
Did it